internal/kubernetes/kubernetes.go

```diff
@@ -196,32 +212,38 @@ func (client *KubeClient) hasAnnotation(annotations map[string]string, status co
 	return false
 }
 
-func (client *KubeClient) loadSecrets(namespace string, secrets []corev1.LocalObjectReference) ([]byte, bool, error) {
-	// TODO: Support all secrets which are referenced as imagePullSecrets instead of only the first one.
+func (client *KubeClient) loadSecrets(namespace string, secrets []corev1.LocalObjectReference) []KubeCreds {
+	allImageCreds := []KubeCreds{}
+
 	for _, s := range secrets {
 		secret, err := client.Client.CoreV1().Secrets(namespace).Get(context.Background(), s.Name, meta.GetOptions{})
 		if err != nil {
-			return nil, false, err
+			logrus.WithError(err).Errorf("Error in loadSecrets: namespace: %s, secret '%s'! Error: %s", namespace, s.Name, err)
+			continue
```

Comment:
Please use `%w` for the last placeholder in the message-template (for the `err`).

Reply:
done